<?xml version="1.0"?>

<launch>
    <node pkg="dji_sdk" type="dji_sdk_node" name="dji_sdk" output="screen">
    <!-- node parameters -->
    <param name="serial_name" type="string" value="/dev/ttyUSB0"/>
    <param name="baud_rate" type="int" value="921600"/>
    <param name="app_id" type="int" value="1022384"/>
    <param name="app_version" type="int" value="1"/>
    <param name="align_time" type="bool" value="false"/>
    <param name="enc_key" type="string" value="e7bad64696529559318bb35d0a8c6050d3b88e791e1808cfe8f7802150ee6f0d"/>
    <param name="use_broadcast" type="bool" value="false"/>
    </node>

    <node pkg="geodetic_utils" type="geodetic_to_local_conversion_node" name="geodetic_to_local_conversion_node" output="screen">
	<remap from="gps" to="dji_sdk/gps_position"/>
	<remap from="imu" to="dji_sdk/imu"/>
	<remap from="vel" to="dji_sdk/velocity"/>
	<remap from="external_altitude" to="dji_sdk/height_above_takeoff"/>
        <remap from="odometry" to="dji_sdk/odometry"/>
    </node>

</launch>
